main.tf

```
@@ -18,7 +18,7 @@ resource "cosmic_instance" "default" {
 
 resource "cosmic_disk" "default" {
   for_each           = var.disks
-  name               = "${var.name}-${each.key}"
+  name               = "${each.key}"
```

We don't need to quote these anymore, these days doing so will create a warning when running a plan or apply.

```suggestion
  name               = each.key
```